packages/documentation/src/stories/getting-started/getting-started.docs.mdx

```diff
@@ -0,0 +1,109 @@
+import { Meta, Source } from '@storybook/blocks';
+import * as GettingStartedStories from './getting-started.stories';
+
+<Meta of={GettingStartedStories} />
+
+# Getting started
```

Comment:
"Getting Started/ Getting Started"
Feels awkward (especially in the address bar), but I couldn't find better